javascript/javascript1/week4/classExercises/creditCardInfo.js

```diff
@@ -0,0 +1,48 @@
+function isValid(arr) {
+  const reversed = arr.reverse();
+  let sumMulti = 0;
+  let sumRest = 0;
+  let sum = 0;
+  for (let i = 1; i < reversed.length; i = i + 2) {
+    doubled = reversed[i] * 2
+    if (doubled >9 ){
+      doubled -= 9;
+    }
+    sumMulti += doubled;
+  }
+  for (let i = 0; i < reversed.length; i = i + 2) {
+    sumRest =  sumRest+ reversed[i];
+  }
+  
+  sum = sumMulti + sumRest;
+  if (sum % 10 === 0) {
+    return true;
+  }
+  return false;
+}
+
+function getCardInfo(num) {
+  const digits = String(num).split("").map(Number);
+  if (isValid(digits)) {
+    if (
+      digits.length === 15 &&
```

Comment:
You've used `digits.length` many times here. A good practice would be to store it in a variable and use it through it instead.